examples/api-usage/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I appreciate the motivation behind adding an `examples/` top level folder;  it may have been a poor choice to introduce a `perspective-examples` module in the project initially, but IMO having 2 is confusing and we should either use `perspective-examples` or move it here, not both.

Reply:
I agree that we shouldn't have both, current the `perspective-examples` package is published, and I did not want to move the publishable packages out of the `packages/` directory. I'd be happy to refactor the examples to be more standalone, but I think this would also break a lot of the tests.